mkdocs.yml

```diff
@@ -16,16 +16,14 @@ nav:
   - Getting started:
       - Polaris: index.md
       - Quickstart: quickstart.md
+      - Resources: resources.md
   - Tutorials:
-      - Basics: tutorials/basics.ipynb
-      - Data Models: tutorials/custom_dataset_benchmark.ipynb
-      - Creating Datasets:
-          - Zarr Datasets: tutorials/dataset_zarr.ipynb
-          - PDB Datasets: tutorials/dataset_pdb.ipynb
-          - SDF Datasets: tutorials/dataset_sdf.ipynb
-          - Optimization: tutorials/optimization.ipynb
-      - Competitions:
-          - tutorials/competition.participate.ipynb
+      - Submit:      
+        - Submit to a Benchmark: tutorials/submit_to_benchmark.ipynb
+        - Submit to a Competition: tutorials/submit_to_competition.ipynb
+      - Create:
```

Comment:
Same here, a more descriptive name.

Reply:
Discussed with Lu over Slack. We can't think of a better alternative and will leave it as is for now. Open to suggestions!